html_css/js/storage.js

```diff
@@ -0,0 +1,68 @@
+export default class Storage {
+  constructor() {
+  }
+
+  getSliderData() {
+    this.initStorage();
+
+    return this.data = JSON.parse(localStorage.getItem('localStorageSliderData'));
```

Comment:
It`s better to save storage key like class property

Also, why do you need this.data here? You are not using it anywhere

Use just:

return JSON.parse(localStorage.getItem('localStorageSliderData'));

If you want to define some property, do it separately

Reply:
Done